assets/src/js/query-no-posts.js

```diff
@@ -2,8 +2,10 @@ export const removeNoPostText = () => {
   const postTitle = document.querySelector('.p4-query-loop .wp-block-heading');
   const postDescription = document.querySelector('.p4-query-loop p');
 
-  if (postTitle.innerHTML === '' && postDescription.innerHTML === '') {
-    const noResultsText = document.querySelector('.p4-query-loop .wp-block-query-no-results');
-    noResultsText.style.display = 'none';
+  if (!postTitle || !postDescription || postTitle.innerHTML !== '' || postDescription.innerHTML !== '') {
```

Comment:
Can't we simplify it a bit like this?
```suggestion
  if (postTitle?.innerHTML !== '' || postDescription?.innerHTML !== '') {
```

Reply:
We absolutely can :D thanks!